.github/workflows/ci.yml

```diff
@@ -109,7 +109,11 @@ jobs:
       - name: Install Cargo Make
         uses: davidB/rust-cargo-make@v1
         with:
-          version: "0.36.0"
+          version: "0.36.0"      
+      - name: Install Protoc
```

Comment:
Will this require devs to have protoc installed on their machines to build the repo? If so, that should be added to the CONTRIBUTING.md and README.md steps.

Reply:
yes it will, I will add the steps in the md files